### ISSUE TITLE
Order doesn't quite work for > 10 bubbles

### ISSUE THREAD

**Issue by vince-helmsdeep** [alexandersimoes/d3plus#417]
Hi Dave, great work btw. 

Trying to use the Bubbles viz. If I have 10 parent bubbles (each with nested bubbles), I can order the parent bubbles from large to small (vice versa). But once I hit 11 and up, they do not order as desired. Any idea why? 

Thanks.

**Comment by davelandry** [comment]
Interesting, can you post a code snippit? Maybe a jsfiddle?

**Comment by vince-helmsdeep** [comment]
Hmm...I can't seem to replicate that. :( 

But the issue now exists for less than 10. 

`  var view3 = [
{"count":19, "name":"f", "group":"A"},
{"count":1, "name":"c", "group":"B"},
{"count":10, "name":"b", "group":"B"},
{"count":67, "name":"f", "group":"B"},
{"count":8, "name":"a", "group":"B"},
{"count":5, "name":"f", "group":"C"},
{"count":4, "name":"f", "group":"D"},
{"count":1, "name":"a", "group":"D"},
{"count":1, "name":"a", "group":"E"}]

	var visualization = d3plus.viz()
	    .container("#viz")     // container DIV to hold the visualization
	    .data(view3)     // data to use with the visualization
	    .type("bubbles")       // visualization type
	    .id(["group","name"]) // nesting keys
	    .depth(1)              // 0-based depth
	    .size("count") // key name to size bubbles
		.order({"sort":"desc"})
	    .color("name") // color by each group
	    .draw()                // finally, draw the visualization!`

I want to sort the biggest bubble to smallest. But the order doesn't work.

**Comment by vince-helmsdeep** [comment]
Ok, here's why. 

When I set the nesting ids  as ["group","name"] and use .order later on, I am defining the order based on "group", in this case, it is alphabetical. 

I want to be able to sort based on the aggregated counts. 

So what I did was create this array:

`var view3 = [
{"total":"19","count":19, "name":"f", "group":"A"},
{"total": "86","count":1, "name":"c", "group":"B"},
{"total":"86","count":10, "name":"b", "group":"B"},
{"total":"86","count":67, "name":"f", "group":"B"},
{"total":"86","count":8, "name":"a", "group":"B"},
{"total":"5","count":5, "name":"f", "group":"C"},
{"total":"5","count":4, "name":"f", "group":"D"},
{"total":"5","count":1, "name":"a", "group":"D"},
{"total":"5","count":1, "name":"a", "group":"E"}];`

`	var visualization = d3plus.viz()
	    .container("#viz")     // container DIV to hold the visualization
	    .data(view3)     // data to use with the visualization
	    .type("bubbles")       // visualization type
	    .id(["total","name"]) // nesting keys
	    .depth(1)              // 0-based depth
	    .size("count") // key name to size bubbles
	    .order({"sort":"asc"})
	    .color("name") // color by each group
	    .draw()                // finally, draw the visualization!

`

The above works for sorting the aggregated bubbles from large to small. However...it breaks down beyond 10 bubbles.  Hope this makes sense.
